element_deeplabcut/dlc.py

```diff
@@ -129,27 +129,42 @@ class File(dj.Part):
 class ModelTrainingParamSet(dj.Lookup):
     definition = """
     # Parameters to specify a DLC model training instance
-    paramset_idx    : smallint
+    # For DLC ≤ 2.0, include scorer_lecacy = True in params
+    paramset_idx                  : smallint
     ---
     paramset_desc: varchar(128)
-    param_set_hash  : uuid        # hash identifying this parameterset
+    param_set_hash                : uuid      # hash identifying this parameterset
     unique index (param_set_hash)
-    params: longblob  # dictionary of all applicable parameters
+    params                        : longblob  # dictionary of all applicable parameters
     """
 
     required_parameters = ('shuffle', 'trainingsetindex')
     skipped_parameters = ('project_path', 'video_sets')
 
     @classmethod
-    def insert_new_params(cls, paramset_desc: str, params: dict, paramset_idx: int = None):
+    def insert_new_params(cls, paramset_desc: str, params: dict,
+                          paramset_idx: int = None, skip_duplicates=False):

```

Comment:
why do we need `skip_duplicates` here? 
Duplicated entries are already handled in this function
+ if the new entry is a duplicate that is identical to existing entry, this would skip
+ if the new entry has a duplicated name but different parameters, errors will be throw
+ if the new entry has different name but identical parameters, a informative statement will be printed